lib/measured/conversion_table.rb

```diff
@@ -31,9 +32,6 @@ def find_conversion(units, to:, from:)
   def find_direct_conversion(units, to:, from:)
     units.each do |unit|
       return unit.conversion_amount if unit.name == from && unit.conversion_unit == to
-    end
-
-    units.each do |unit|
```

Comment:
Interesting. This is a subtle change in algorithm. So the performance improvement may be that change, not the extra iteration. But either way, I don't think it's returning a wrong result.

Reply:
That's a good point! I took that for granted 😄 .